ui/perfherder/alerts/Pagination.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+import withValidation from '../Validation';
+
+class PaginationGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      page: this.props.page,
+      count: 0,
+    };
+  }
+
+  navigatePage = page => {
+    this.setState({ page }, this.props.fetchData);
+    this.props.updateParams({ page });
```

Comment:
Since you're typing `this.props...` twice, you can destructure the object values like so: 
` const { fetchData, updateParams } = this.props;`